packages/cms-cli/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  testPathIgnorePatterns: ['commands/functions/test.js'],
```

Comment:
Not a big deal but what is this for?  Was it breaking jest?

Reply:
Jest automatically looks at `test.js` files as tests and tries to run them, so had to add this to prevent that.